code/modules/surgery/organs/blood.dm

```diff
@@ -96,7 +96,7 @@
 		if(dna.species.exotic_blood)
 			var/datum/reagent/R = GLOB.chemical_reagents_list[get_blood_id()]
 			if(istype(R) && isturf(loc))
-				R.reaction_turf(get_turf(src), amt * EXOTIC_BLEED_MULTIPLIER)
+				R.reaction_turf(get_turf(src), amt * EXOTIC_BLEED_MULTIPLIER, dna.species.blood_color)
```

Comment:
This is fairly tightly coupled to the implementation of slimejelly now. Any other exotic blood type now also has to have this implementation.
I don't see a good alternative quickly though

Reply:
Made it check for an exotic_colour trait to fix the issue